pkgs/development/compilers/llvm/common/compiler-rt/default.nix

```diff
@@ -131,6 +132,8 @@ stdenv.mkDerivation ({
     "-DCOMPILER_RT_ENABLE_IOS=OFF"
   ]) ++ lib.optionals (lib.versionAtLeast version "19" && stdenv.isDarwin && lib.versionOlder stdenv.hostPlatform.darwinMinVersion "10.13") [
     "-DSANITIZER_MIN_OSX_VERSION=10.10"
+  ]  ++ lib.optionals (noSanitizers && lib.versionAtLeast release_version "20") [
```

Comment:
Should be at least 19 since [this build](https://hydra.nixos.org/build/268346779) fails.

Reply:
Done!